src/popup/popup.js

```diff
@@ -286,18 +286,27 @@ elements.searchIcon.addEventListener('click', () => {
     });
   elements.clearSearchButton[0].classList.remove('display-none');
 });
-
+elements.modal.addEventListener('click', () => {
+  elements.modalCancel.click();
+});
```

Comment:
This will close the modal even if we click on it. If the user, while trying to click the "YES" button, accidentally clicks at the side, it will be closed. This is not what we want.